README.md

```diff
@@ -90,23 +91,23 @@ training:
     loss:
         name: BCELoss
         kwargs: {reduction: "mean"}
-        transformations: 
-            - Sigmoid
+        transformations:
+            - {name: Sigmoid}
     optimizer:
         name: Adam
-        kwargs: {beta: 0.01}
+        kwargs: {lr: 0.002}
     pretrained_on:
-        - {source: doi1, hash: 21}
+        - {name: NucleiDataset}  # optional: source : {identifier: doi/url, hash: hash_value}
```

Comment:
maybe we can add the number of epochs trained here... This might be a useful metric, especially when training on several different datasets to get an idea of their influence.

Reply:
Good point.